assets/templates/messages-header.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Doppler UI Team</title>
+  <meta name="keywords" content="">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="canonical" href="" />
+  <!-- Open Graph metatags -->
+  <meta property="og:title" content="" />
+  <meta property="og:type" content="article" />
+  <meta property="og:image" content="" />
+  <meta property="og:url" content="" />
+  <meta property="og:description" content="" />
+  <meta property="og:site_name" content="" />
+  <!-- Stylesheets and polyfills -->
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <!-- Favicons -->
+  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.fromdoppler.com/apple-touch-icon-144x144-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.fromdoppler.com/apple-touch-icon-114x114-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.fromdoppler.com/apple-touch-icon-72x72-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="https://www.fromdoppler.com/apple-touch-icon-57x57-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" href="https://www.fromdoppler.com/apple-touch-icon-precomposed.png" />
+  <link rel="apple-touch-icon" href="https://www.fromdoppler.com/apple-touch-icon.png" />
+  <link rel="icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
+  <link rel="shortcut icon" type="image/x-icon" href="https://www.fromdoppler.com/favicon.ico" />
+</head>
+<body>
+<!-- rojo -->
+<div class="messages-container error">
+  <div class="wrapper">
+    <p class="ng-binding">Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.</p>
+    <a href="#" class="button button--light button--tiny">Verificar Ahora</a>
+  </div>
+</div><!-- error -->
+<!-- amarillo -->
+<div class="messages-container warning">
+  <div class="wrapper">
+    <p>Thanks! We’ll contact you ASAP and if everything is fine, you’ll be able to send your Campaign.</p>
+  </div>
+</div><!-- warning-->
+<!-- verde -->
+<div class="messages-container message">
```

Comment:
can add css class 'info' with the same styles that 'message'. We need for the webapp. Thanks!